test-crates/cargo-mock/src/main.rs

```diff
@@ -33,7 +33,8 @@ fn run() -> Result<()> {
         .replace("--quiet", "")
         .replace(&cwd, "")
         .replace(" ", "-")
-        .replace("/", "-");
+        .replace("/", "-")
+        .replace("-----C-link-arg=-s", "");
```

Comment:
Why do we need to do this? I don't quite understand why it starts with so many `-`s.

Reply:
I think it's `" -- -C link-arg=-s".replace(' ', '-')`. The string got too long for a filename so' i'm cutting stuff out; this could be encoded much nicer but i think for an optional test dependency it's not worth it